src/Serehfa/Arithmetics.cs

```diff
@@ -7,6 +7,12 @@ namespace Serehfa
 
     public static class Arithmetics
     {
+		[LispBuiltin("zero?")]
```

Comment:
This indentation wackiness is Emacs. Looks like were' missing a `.editorconfig` file for this repo.